apps/web/src/components/milestone-logs/Recruitment.tsx

```diff
@@ -43,11 +43,20 @@ export const Recruitment: React.FC = () => {
           (job.job_title && filters.job_title.includes(job.job_title.id)),
       )
       .slice(pageSize * (pageIndex - 1), pageSize * pageIndex);
+
     setJobRecords(jobs || []);
 
-    setTotal(applicant?.jobs?.length || 0);
+    // check if results should show filtered jobs or total jobs
+    isFiltered(filters) ? setTotal(jobs?.length || 0) : setTotal(applicant?.jobs?.length || 0);
```

Comment:
jobs(not applicant.jobs) are ones in a page. don't we need another variable?
- all jobs
- filtered jobs -> to be used to set total
- jobs in the current page